assets/.eslintrc.json

```diff
@@ -1,5 +1,8 @@
 {
   "env": {
     "browser": true
+  },
+  "rules": {
+    "strict": "off"
```

Comment:
Weird, I'd expect it to handle the browser case. How about `strict: [2, "function"]`? Can you test if it works OK for both cases? Otherwise there is the `safe` option too.